.vscode/extensions.json

```diff
@@ -1,6 +1,7 @@
 {
   "recommendations": [
     "biomejs.biome",
-    "esbenp.prettier-vscode"
+    "esbenp.prettier-vscode",
+    "sysoev.vscode-open-in-github"
```

Comment:
not sure if should go as far and add this here, might be better to only list essential extensions here

Reply:
Without adding that extension, it's settings `openInGitHub.defaultBranch` will not be available. Also it's a recommended list, not compulsory for everyone.

Reply:
not sure either.  there are a few that are super helpful like "npm intellisense" and "path intellisense" but its opening a can of worms...

Reply:
@matthewkeil If you think any extension is useful for other team members and community who is contributing to this project, please share.

Reply:
honestly i hate when repos bring up warning when i dont have an extension installed. its really annoying...i think only ones that are ABSOLUTELY necessary, like for the lint command, that should be included

Reply:
For that I would recommend VSCode to have two workspace settings file. One `.vsode/settings.json` to shared among users and other `.vscode/settings.local.json` only for individual use.

Reply:
@matthewkeil @nflaig So should I keep this plugin and related settings? or remove it from this PR?

Reply:
> repos bring up warning when i dont have an extension installed.

that's my concern as well, not sure how often vs code will show the popup but better to keep this list to essential extensions

> So should I keep this plugin and related settings?

Keep the setting, remove the recommendation. Could add a comment in the setting file for which extension it is, people can manually install it if they want to

Reply:
@nflaig Can't use comments for the `.json` files, and as per my knowledge VSCode not yet supporting `.jsonc` format.